packages/kbn-code-owners/src/file_code_owner.ts

```diff
@@ -87,8 +86,11 @@ export async function runGetOwnersForFileCli() {
       if (!targetFile) throw createFlagError(`Missing --file argument`);
       existOrThrow(targetFile); // This call is duplicated in getPathsWithOwnersReversed(), so this is a short circuit
       const result = getCodeOwnersForFile(targetFile);
-      if (result) log.success(result);
-      else log.error(`Ownership of file [${targetFile}] is UNKNOWN`);
+      if (result) {
+        const path = result.path.includes('/', 0) ? trimFrontSlash(result.path) : result.path;
```

Comment:
I think you could just apply the replace and don't have to check for a leading slash upfront.

Reply:
True. In that case, the replace would just do nothing.

Reply:
oh yeah, great argument lol

Reply:
https://github.com/elastic/kibana/pull/202704/commits/d94d653f6eb2e99a744bb415ee69f6425913ed19